samples/bluetooth/mesh/light/README.rst

```diff
@@ -132,17 +132,36 @@ This sample is split into the following source files:
 DFU configuration
 =================
 
-To enable the DFU feature for the supported development kits, set :makevar:`SB_CONF_FILE` to :file:`sysbuild-dfu.conf` and set :makevar:`EXTRA_CONF_FILE` to :file:`overlay-dfu.conf` when building the sample.
-For example, when building from the command line, use the following command:
+.. tabs::
 
-  .. code-block:: console
+   .. tab:: nRF52840 and nRF54L15 DK
```

Comment:
Now I'm nitpicking, but:
```suggestion
   .. tab:: nRF52840 DK and nRF54L15 PDK
```

... and also add `DK` to the other tab to align with this.